### ISSUE TITLE
Serverless Stack tutorial showing use of Angular instead of React, with hosting in AWS using serverless-s3-sync instead of netlify

### ISSUE THREAD

**Issue by GurpreetSingh5** [AnomalyInnovations/serverless-stack-com#399]
Serverless Stack tutorial showing use of Angular instead of React, with hosting in AWS using serverless-s3-sync instead of netlify for a purely AWS services based development stack experience ?
Any plans for that?
If not, I'm willing to contribute, as I have certain degree of experience with it.

**Comment by jayair** [comment]
Hi there, so we do have a section on using S3 for hosting your React app. But with the `serverless-s3-sync` plugin, you would have to put your React code in your Serverless app right?

**Comment by GurpreetSingh5** [comment]
I had enquired regarding using Angular instead of React.

**Comment by jayair** [comment]
Yeah we just don't get enough requests for Angular sadly. But if you put together a sample repo with some instructions on how to build the notes app with Angular, we'll share it with the community.
